subprojects/gdk_rust/gdk_common/src/model.rs

```diff
@@ -148,7 +148,12 @@ pub struct GetAddressOpt {
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct CreateAccountOpt {
+    pub subaccount: u32,
     pub name: String,
+}
```

Comment:
Integration tests needs updating, `CreateAccountOpt` does not have `script_type` field.

Reply:
Addressed by https://github.com/Blockstream/gdk/pull/102/commits/cb8d1b1eb6e8c16cd3397816e2d91c21f4c67ef7.